src/config/firefox-desktop.js

```diff
@@ -43,6 +43,13 @@ export default {
       },
     },
   },
+  transformProbeForGLAM(probe) {
```

Comment:
This is a good solution but I still wish it wasn't necessary.

Reply:
Perhaps that's a change that could be pushed to probe-scraper, to ultimately end up in probe-info?

Reply:
I think that would be ideal. Do you know how to go about requesting that @robhudson? Is the probe scraper versioned?

Reply:
By the way, there's also `"all_childs"`: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/collection/histograms.html#record-in-processes

Reply:
`"all_childs"` -> `['content', 'gpu']` (all except `'main'`)

Reply:
https://github.com/mozilla/probe-scraper/issues/187